src/Forms/ConfirmedPasswordField.php

```diff
@@ -255,7 +267,33 @@ public function getChildren()
     public function setCanBeEmpty($value)
     {
         $this->canBeEmpty = (bool)$value;
+        $this->updateRightTitle();
+        return $this;
+    }
+
+    public function getGenerateRandomPasswordOnEmpty(): bool
+    {
+        return $this->generateRandomPasswordOnEmpty;
+    }
 
+    public function setGenerateRandomPasswordOnEmpty(bool $generateRandomPasswordOnEmpty): static
+    {
+        $this->generateRandomPasswordOnEmpty = $generateRandomPasswordOnEmpty;
```

Comment:
```suggestion
    public function setGenerateRandomPasswordOnEmpty(bool $generate): static
    {
        $this->generateRandomPasswordOnEmpty = $generate
```
The purpose of the variable is obvious, so lets keep its name short. This is especially important since arguments can be passed like `$field->setGenerateRandomPasswordOnEmpty(generateRandomPasswordOnEmpty: true);` which is way too verbose compared to `$field->setGenerateRandomPasswordOnEmpty(generate: true);`

Reply:
Updated